app/(dashboard)/mailboxes/[mailbox_slug]/settings/team/teamSetting.tsx

```diff
@@ -22,6 +35,19 @@ const TeamSetting = ({ mailboxSlug }: TeamSettingProps) => {
     >
       <div className="w-full space-y-6">
         <AddMember mailboxSlug={mailboxSlug} teamMembers={teamMembers} />
+
+        {teamMembers.length > 0 && (
+          <div className="relative">
+            <Search className="absolute left-3 top-1/2 transform -translate-y-1/2 h-4 w-4 text-muted-foreground" />
+            <Input
+              placeholder="Search team members by name, email, or keywords..."
```

Comment:
```suggestion
              placeholder="Search..."
```

I don't think we need to specify since it's every field